shared/actions/settings.js

```diff
@@ -176,29 +177,46 @@ function* saveNotificationsSaga(): SagaGenerator<any, any> {
     }
 
     let JSONPayload = []
+    let chatGlobalArg = {}
     for (const groupName in current.groups) {
       const group = current.groups[groupName]
-      for (const key in group.settings) {
-        const setting = group.settings[key]
+      if (groupName === Constants.securityGroup) {
+        // Special case this since it will go to chat settings endpoint
+        for (const key in group.settings) {
+          const setting = group.settings[key]
+          chatGlobalArg[`${ChatTypes.CommonGlobalAppNotificationSetting[setting.name]}`] = setting.subscribed
```

Comment:
That happens on the Go side. I wanted to use the same type in AVDL for these settings, but the keys are basically ints, so I couldn't just pass them in directly from JS. Instead I pass in the string version of the value and convert in Go.